crates/core/src/rpc/accounts_data.rs

```diff
@@ -122,7 +149,10 @@ impl AccountsData for SurfpoolAccountsDataRpc {
             .iter()
             .map(|s| {
                 let pk = verify_pubkey(s)?;
-                transform_account_to_ui_account(&state_reader.svm.get_account(&pk), &config)
+                match state_reader.svm.get_account(&pk) {
```

Comment:
Shouldn't we be hitting RPC here too?

Reply:
Yeah, we haven't done this method yet, this was just making the compiler happy